test/e2e/lua/dynamic_configuration.go

```diff
@@ -70,6 +70,30 @@ var _ = framework.IngressNginxDescribe("Dynamic Configuration", func() {
 		})
 	})
 
+	Context("Lua shared dict", func() {
```

Comment:
Why do you need this test when you have https://github.com/kubernetes/ingress-nginx/pull/4405/files#diff-a7c5569c149942c4c70abad3c1fcb40fR38?